test/utils.py

```diff
@@ -711,6 +711,10 @@ def initdb(self):
             # Make sure this is consistent across platforms
             pgconf.write("datestyle = 'iso, mdy'\n")
 
+            # Make PostgreSQL listen on both IPv4 and IPv6 (if supported)
+            if HAVE_IPV6_LOCALHOST:
+                pgconf.write("listen_addresses='0.0.0.0,::'\n")
```

Comment:
This should use the loopback addresses

```suggestion
                pgconf.write("listen_addresses='127.0.0.1,::1'\n")
```

Reply:
okay, that works too. Accepting your suggested change.